### DISCUSSION TITLE
exit Klipper screen

### DISCUSSION THREAD

**Discussion by keshka** [KlipperScreen/KlipperScreen#1500] [Q&A] [answered]
My Pi has a full desktop environment. Once logged in via a local touch screen, I often run the Klipper Screen app. I there a way to exit the app and return to the desktop. The only method I have available at this time is using Klipper Screen to reboot the system.

**Comment by alfrix** [comment, marked as answer]
when running as an application the window will have a titlebar with the X

alternatively you can install https://github.com/dw-0/kiauh/blob/master/docs/gcode_shell_command.md
and create a macro with a command that kills the app

but you should not run klipperscreen as a service, if you installed as a service you need to disable the service
